testsuite/integration/microprofile-tck/rest-client/pom.xml

```diff
@@ -0,0 +1,275 @@
+              <params>--port=8765</params>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+           <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>install-as-module-wiremock</id>
+                        <phase>generate-test-resources</phase>
+                        <goals>
+                            <goal>copy</goal>
+                        </goals>
+                        <configuration>
+                            <artifactItems>
+                                <artifactItem>
+                                    <groupId>com.github.tomakehurst</groupId>
+                                    <artifactId>wiremock</artifactId>
+                                </artifactItem>
+                                <artifactItem>
+                                    <groupId>junit</groupId>
+                                    <artifactId>junit</artifactId>
+                                </artifactItem>
+                            </artifactItems>
+                            <stripVersion>true</stripVersion>
+                            <outputDirectory>${project.build.directory}/wildfly/wiremock</outputDirectory>
+                         </configuration>
+                     </execution>
+                 </executions>
+           </plugin>
+           <plugin>
+                 <groupId>org.wildfly.plugins</groupId>
+                 <artifactId>wildfly-maven-plugin</artifactId>
+                 <version>${version.org.wildfly.plugin}</version>
+                 <executions>
+                     <execution>
+                        <id>add-wiremock-module</id>
+                        <phase>process-test-resources</phase>
+                        <goals>
+                            <goal>execute-commands</goal>
+                        </goals>
+                        <configuration>
+                        <commands>
+                            <command>module add --name=com.github.tomakehurst.wiremock --resources=${project.build.directory}/wildfly/wiremock/wiremock.jar:${project.build.directory}/wildfly/wiremock/junit.jar --module-xml=${basedir}/module.xml</command>
+                        </commands>
+                        <scripts>
+                            <script>wiremock.cli</script>
+                        </scripts>
+                        </configuration>
+                     </execution>
+                  </executions>
+                  <configuration>
+                      <offline>true</offline>
+                      <jboss-home>${basedir}/target/wildfly</jboss-home>
+                      <stdout>${project.build.directory}/wildfly-wiremock.log</stdout>
+                      <java-opts>${modular.jdk.args}</java-opts>
+                      <system-properties>
```

Comment:
@asoldano 

add

<maven.repo.local>${settings.localRepository}</maven.repo.local>

Reply:
for example 

https://github.com/wildfly/wildfly/blob/master/testsuite/integration/clustering/pom.xml#L163
https://github.com/wildfly/wildfly/blob/master/testsuite/compat/pom.xml#L341
https://github.com/wildfly/wildfly/blob/master/testsuite/integration/manualmode/pom.xml#L166

etc